### DISCUSSION TITLE
How to make a surface on a cylinder?

### DISCUSSION THREAD

**Discussion by teajaysee** [pyvista/pyvista#6416] [Q&A]
### First check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the PyVista documentation, with the integrated search.
- [X] I already searched in Google "How to X in PyVista" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I will NOT make duplicate discussions or issues in the future.

### Commit to Help

- [X] I commit to help with one of those options.

### Sample Code What is the problem, question, or error?
Write a short description telling me what you are doing, what you expect to happen, and what is currently happening.

```python
xs = []
ys = []
zs = []
for h in range(24): 
 for y in range(100): 
  xs.append (h) 
  ys.append(y) 
  zs.append(10 * np.abs(y-100)/50  * np.abs (h-12)/12) 
points = np.c_[np.array(xs), np.array (ys), np.array (zs) ] 
cloud = pv.PolyData(points) 
surf = cloud.delaunay2d() 
surf.plot (show_edges=True)
```

### Description

I am plotting a surface similar to below where one of the variables is the hour of the day.
What I would like to do is wrap the surface round so hour 23 joins hour 0 and the surface becomes like a cylinder.
Can anyone help me how i do this?
The hour can also be expressed as an angle, or as two variables, sin and cos 2pi*hour/24 
Many thanks for any help,

xs = []
ys = []
zs = []
for h in range(24): 
 for y in range(100): 
  xs.append (h) 
  ys.append(y) 
  zs.append(10 * np.abs(y-100)/50  * np.abs (h-12)/12) 
points = np.c_[np.array(xs), np.array (ys), np.array (zs) ] 
cloud = pv.PolyData(points) 
surf = cloud.delaunay2d() 
surf.plot (show_edges=True) 

### System Information

```shell
No bug
```

**Comment by MatthewFlamm** [comment]
Does [CylinderStructured](https://docs.pyvista.org/api/utilities/_autosummary/pyvista.cylinderstructured#pyvista.CylinderStructured) do what you want?

Otherwise, you can create a `PolyData` object with points and faces if you just want the outer surface, i.e. it is 2D, or a different type of object if that object fills 3D space.  An example for `PolyData` is given https://docs.pyvista.org/examples/00-load/create-poly#sphx-glr-examples-00-load-create-poly-py.
